persistence/pg.go

```diff
@@ -82,6 +86,19 @@ func unmarshallDbInvoice(invoice *dbInvoice) *Invoice {
 	}
 }
 
+func DBInvoiceToProto(invoice *dbInvoice) *lnmuxrpc.Invoice {
```

Comment:
Maybe this can remain private, as we don't want to expose `dbInvoice` anyway.

Reply:
Actually, we use DBInvoiceToProto in server.go
We need to export this function

Otherwise we need to return a []lnmuxrpc.Invoice in `persistence` package but it doesn't make sense IMO?

or I can move this function in server.go, but I thought that it should be in the persistence package because we are working on a `dbInvoice`

Reply:
Can't we just return `[]*pg.Invoice`?

Reply:
you mean for `func (p *PostgresPersister) GetSettledInvoices()` call?

Reply:
yes

Reply:
Ah yeah we can, so I can move the proto conversion in server.go